splunk_connect_for_snmp_traps/manager/hec_sender.py

```diff
@@ -32,23 +32,17 @@ def get_session(self):
         return self._thread_local.session
 
     def post_data_to_thread_pool(self, host, variables_binds):
-        headers = {
-            "Authorization": f"Splunk {self._hec_config.get_authentication_token()}"
-        }
         data = {
             "sourcetype": "sc4snmp:traps",
             "host": host,
             "index": self._args.index,
-            # "index": self._server_config["splunk"]["index"],
             "event": variables_binds,
         }
+
         try:
             session = self.get_session()
-            for endpoint in self._hec_config.get_endpoints():
-                response = session.post(
-                    url=endpoint, json=data, headers=headers, verify=False
-                )
-                logger.debug(f"Response code is {response.status_code}")
+            response = session.post(url=self._endpoint, json=data)
```

Comment:
I think we need to set verify=True/False based on the value of the environment value we pass?

Reply:
We use HTTP to connect to OTEL from both trap and poller. This flag will be only interpreted in OTEL as it is using HTTPS for outgoing transmission.